static/js/apps/biomed_nl/app.tsx

```diff
@@ -412,6 +455,19 @@ export function App(): ReactElement {
                         Tell us how we did.
                       </a>
                     </p>
+                    <br></br>
+                    <p>
+                      <span
+                        onClick={resetToSampleQueries}
+                        css={css`
+                          color: ${theme.colors.link.primary.base};
+                          cursor: pointer;
+                        `}
+                      >
+                        &larr; Back
+                      </span>
+                      &nbsp;to see sample queries.
```

Comment:
nit: I'm not sure how I feel about the color change between back and the rest of the string .. what if we just do "Back to sample queries" all as a link?

Reply:
done!

![image](https://github.com/user-attachments/assets/b9b13d18-6cf4-4ac7-bb1f-c8389c9cad16)